AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -398,4 +394,11 @@ + (int)setMaxPageCount:(long)maxPageCount inOpenedDatabase:(void *)db {
   return [MSDBStorage executeNonSelectionQuery:statement inOpenedDatabase:db];
 }
 
++ (void)load {
```

Comment:
This method isn't used.

Reply:
This method will be invoked before the class initializes

Reply:
I think he implies [this](https://developer.apple.com/documentation/objectivec/nsobject/1418815-load?language=objc). Looks good to me now when I remembered this one

Reply:
Yes the method is called by the runtime when the class is loaded. The SDK doesn't need to call it.

Reply:
OK. Good to know. 😄